### DISCUSSION TITLE
Export PDF to PNG or JPG

### DISCUSSION THREAD

**Discussion by cassiorezende** [pdfarranger/pdfarranger#1145] [Ideas]
Would it be possible to add the function to export pages as PNG or JPG images (perhaps using pdftoppm)?

**Comment by dreua** [comment]
Duplicate: https://github.com/pdfarranger/pdfarranger/issues/1090

**Reply by dreua** [reply]
Feel free to use the :+1: reaction on #1090 to show interest and Subscribe to get notifications in case there should be any development.

**Reply by mak448a** [reply]
Please consider reopening this. (workaround: use GIMP to convert the pdf to a png)

**Reply by dreua** [reply]
@mak448a See my comment on your duplicate comment on the duplicate linked issue, thanks :smile:
